ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/layout.tsx

```diff
@@ -47,6 +50,20 @@ export default function FunctionLayout({ children, params }: FunctionLayoutProps
       icon: <CommandLineIcon className="w-3.5" />,
     },
   ];
+
+  if (isReplayEnabled) {
```

Comment:
This will always be true since `useBooleanFlag` returns an object. Ideally `useBooleanFlag` would return a boolean but it needs to consider that the user may not be initialized yet

Reply:
Maybe we should enable this rule:
https://typescript-eslint.io/rules/strict-boolean-expressions